src/style.css

```diff
@@ -1,3 +1,175 @@
+html {
+  --acdc-color: #00acdc;
```

Comment:
create variables with root selector
```suggestion
  --main-color: #00acdc;
```